Dwolla.Client/HttpServices/CustomersHttpService.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Dwolla.Client.HttpServices.Architecture;
+using Dwolla.Client.Models.Requests;
+using Dwolla.Client.Models.Responses;
+using Dwolla.Client.Rest;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Dwolla.Client.HttpServices
+{
+	public class CustomersHttpService : BaseHttpService
+	{
+		public CustomersHttpService(IDwollaClient dwollaClient, Func<Task<string>> getAccessToken)
+			: base(dwollaClient, getAccessToken)
+		{
+		}
+
+		public async Task<RestResponse<Customer>> GetCustomerAsync(string customerId, CancellationToken cancellation = default)
+		{
+			if (string.IsNullOrWhiteSpace(customerId))
+			{
+				throw new ArgumentException("CustomerId should not be blank.");
+			}
+
+			return await GetAsync<Customer>(new Uri($"{client.ApiBaseAddress}/customers/{customerId}"), cancellation);
+		}
+
+		public async Task<RestResponse<IavTokenResponse>> GetCustomerIavTokenAsync(string customerId, CancellationToken cancellation = default)
+		{
+			if (string.IsNullOrWhiteSpace(customerId))
+			{
+				throw new ArgumentException("CustomerId should not be blank.");
+			}
+
+			return await GetAsync<IavTokenResponse>(new Uri($"{client.ApiBaseAddress}/customers/{customerId}/iav-token"), cancellation);
+		}
```

Comment:
The IAV token endpoint has now been removed from the API as part of [dwolla.js sunset](https://developers.dwolla.com/changelog#2023-08-31).

```suggestion

```